authentication/migrations/0002_group_migrations.py

```diff
@@ -109,14 +19,14 @@ def populate_groups_with_permissions(apps, schema_editor):
     """
     emit_post_migrate_signal(2, False, "default")
 
-    for group_name in permissions_by_group:
+    for group_name in PERMISSIONS_BY_GROUP:
         group, created = Group.objects.get_or_create(name=group_name)
         if created:
             LOGGER.info('Group "{}" created'.format(group_name))
 
         # TODO: once groups are locked down, only do this when created == True
         perm_list = []
-        for perm_code in permissions_by_group[group_name]:
+        for perm_code in PERMISSIONS_BY_GROUP[group_name]:
```

Comment:
python nit picking - 
`for perm_code in PERMISSIONS_BY_GROUP.get(group_name, [])`

this will return an empty array if groupName isn't. found which won't crash the code. Just minor thing that makes cleaner code.